app/src/main-process/menu/build-default-menu.ts

```diff
@@ -425,32 +425,40 @@ export function buildDefaultMenu({
   const submitIssueItem: Electron.MenuItemConstructorOptions = {
     label: __DARWIN__ ? 'Report Issue…' : 'Report issue…',
     click() {
-      shell.openExternal('https://github.com/desktop/desktop/issues/new/choose')
+      shell
+        .openExternal('https://github.com/desktop/desktop/issues/new/choose')
+        .catch(err => log('error', `Call to openExternal failed: '${err}'`))
```

Comment:
It seems this file erroneously imports the `log` method from `log.ts` which shadows the [globally available `log` object](https://github.com/desktop/desktop/blob/284e374294289fc44d45bd115031027b3ce1f136/app/src/lib/globals.d.ts#L100-L168) (installed [here for the main process](https://github.com/desktop/desktop/blob/284e374294289fc44d45bd115031027b3ce1f136/app/src/lib/logging/main/install.ts) and [here for the renderer](https://github.com/desktop/desktop/blob/284e374294289fc44d45bd115031027b3ce1f136/app/src/lib/logging/renderer/install.ts)).

Once you've removed that import you should get access to `log.error` which takes an error message as its first parameter and an optional exception as its second. By passing the exception separately we can globally decide how to format our error messages in the log file.

I'd be looking for something similar to this

```suggestion
        .catch(err => log.error('Failed opening issue creation page', err))
```